modules/pytket-qiskit/tests/test-requirements.txt

```diff
@@ -1 +1,2 @@
-pytest-timeout ~= 1.4.2 
\ No newline at end of file
+-r ../../../base-test-requirements.txt
```

Comment:
Can we make this a separate PR, changing all extensions at once rather than just this one?
Currently these are installed in the `build-test` script; if we do make this change then we should remove the installation from there.

Reply:
I would prefer this change to be in a separate PR though.